cmd/boulder-wfe2/main.go

```diff
@@ -186,7 +186,14 @@ func loadCertificateFile(aiaIssuerURL, certFile string) ([]byte, *issuance.Certi
 	if pemBytes[len(pemBytes)-1] != '\n' {
 		pemBytes = append(pemBytes, '\n')
 	}
-	return pemBytes, &issuance.Certificate{Certificate: cert}, nil
+	ic, err := issuance.NewCertificate(cert)
+	if err != nil {
+		return nil, nil, fmt.Errorf(
+			"CertificateChain entry for AIA issuer url %q has an "+
+				"invalid chain file: %q - unable to compute hashes over issuer cert",
```

Comment:
I think the last part of this error ("unable to compute hashes over issuer cert") should be generated by issuance.NewCertificate. Also I think the only actual error case is more like "unable to marshal Subject for hashing"

Reply:
Genericized this error message, added the actual error as a wrapped error, and improved the construction of a few of the other errors in this function to match.